lib/matterStacks.ts

```diff
@@ -159,6 +161,10 @@ export class MatterStack extends Stack {
                 type: "String",
                 description: "The Common Name for this PAI"
             }).valueAsString;
+            let crlBucketName = new CfnParameter(this, 'crlBucketName', {
```

Comment:
This can be surprising that bucket needs to exist, what about we simply create a separate bucket for PAIs?

Or we could re-use the PAA's bucket by passing bucket's name via a PAA's tag, or maybe an SSM StringParameter.

Reply:
Updated it so it will make a new bucket for PAIs